frontend/ui/data/keyboardlayouts/keypopup/ar_AA_popup.lua

```diff
@@ -0,0 +1,162 @@
+return {
+    alef = {
+        "ا",
+        north = "أ",
+        northeast = "إ",
+        northwest = "ء",
+        east = "آ",
+        west = "ﭐ",
+    },
+    ba = {
+        "ب",
+        north = "ݒ",
+    },
+    jeem = {
+        "ج",
+        north = "ج",
+    },
+    daal = {
+        "د",
+    },
+    h_aa = { -- الهاء
+        "ه",
+        north = "هـ",
+    },
+    waw = {
+        "و",
+    },
+    zay = {
+        "ز",
+        north = "ژ",
+    },
+    ha = { -- الحاء
```

Comment:
You could translate that arabic word to english      `-- الحاء (english translation)` if that means something other than "ha" :) so we don't have to wonder if we're missing something, and to improve the culture of non arabic readers :)
(I wonder the same above for h_aa, but it's shorter, so it would just be h_aa in arabic :)

Reply:
@poire-z I didn't know how to "transcribe" the two letters, so I used 'ha' for هـ and 'h_aa' for ح. Honestly, this could be improved, and I myself mix up the two "transciptions". 
Please do listen to the first 5 seconds of this YouTube video: https://www.youtube.com/watch?v=YeyVjaWD6-U. The first letter that the announcer pronounces is 'هـ' (my 'ha') and the second letter is 'ح' (my h_aa).

Reply:
The word 'الحاء' is simply the name of the letter, roughly meaning "the letter h_aa". like how "double u" is sort of a name of the letter W in English.